osx/KA-Lite/KA-Lite/README.md

```diff
@@ -1,41 +1,38 @@
 KA-Lite OS X App
-========================
+================
 
 This is the KA-Lite status menu app with the source and PyRun in one package.
```

Comment:
We don't include the "source" anymore, it's part of the pip install done on PyRun.